django/db/models/lookups.py

```diff
@@ -224,13 +224,13 @@ def process_rhs(self, compiler, connection):
     def get_rhs_op(self, connection, rhs):
         return 'IN %s' % rhs
 
-    def as_sql(self, compiler, connection):
+    def as_sql(self, compiler, connection, **kwargs):
         max_in_list_size = connection.ops.max_in_list_size()
         if self.rhs_is_direct_value() and max_in_list_size and len(self.rhs) > max_in_list_size:
-            return self.split_parameter_list_as_sql(compiler, connection)
-        return super(In, self).as_sql(compiler, connection)
+            return self.split_parameter_list_as_sql(compiler, connection, **kwargs)
+        return super(In, self).as_sql(compiler, connection, **kwargs)
 
-    def split_parameter_list_as_sql(self, compiler, connection):
+    def split_parameter_list_as_sql(self, compiler, connection, **kwargs):
```

Comment:
I don't think this method requires `**kwargs` because it's not part of the expressions API, it just looks similar.